src/components/SchemaResourcePage/SchemaResource.tsx

```diff
@@ -25,7 +27,11 @@ export function SchemaResource({
     if (!id) return;
     setLoadingState("loading");
     getResource(resourceInfo, id)?.then((res) => {
-      setResource(res.data[0]);
+      const specData = res.data[0]?.spec;
+      setResource({
+        ...res.data[0],
+        spec: specData && !isEmpty(specData) ? dump(specData) : ""
```

Comment:
can we set sortKeys = true?

Reply:
I did not understand the context. Do you want me to add a property called sortKeys to the resource object?

Reply:
No - js-yaml supports an option called `sortKeys` when dumping

Reply:
Added the option